ethereum_history_api/tests/src/setupAnvil.ts

```diff
@@ -0,0 +1,13 @@
+import assert from 'assert';
+import { ChildProcess, spawn } from 'child_process';
+
+let anvil: ChildProcess;
```

Comment:
How about moving anvil to argument/class?
i.e.
```
setup(anvil)
teardown(anvil)
```

or 
```
anvil.start()
anvil.stop()
```

Global variables/singleton is antypattern :P

Reply:
These names should be like that because it's the vitest way to do it.
Agree that it's an antipattern, but feels justified here. Alternatively I can wrap it in a class and call class methods in those functions, but it will not improve anything